fix(docs): fix a couple of typos in stacks.mdx (#2269)
hashicorp/terraform-cdk: website/docs/cdktf/concepts/stacks.mdx

From a usage perspective it looks like we are accessing the `id` value of `vpc` from the `origin-stack` instance of `VpcStack` and then referencing it in the `target-stack` instance of `BackendStack`.
Accessing a value from a different stack causes the value to be exported as `TerraformOutput` in the origin stack.
The value is then accessed through a `TerraformRemoteState` in the target stack.
Both are automatically added to the respective stacks to make the process seemless.

> When you are using Terraform Cloud it's necessary for each stack to be its own workspace.

Reviewer: ```suggestion
When you are using Terraform Cloud, each stack must be its own workspace.
```

Author: Done. To keep the git history clean, I decided to amend the commit instead of creating a new one by accepting the suggested change. I hope that's OK.

Reviewer 2: Sure!
